src/hyppo/dos.asm

```diff
@@ -947,6 +956,19 @@ dos_d81write_en:
         sec
         rts
 
+dos_get_proc_desc:
+
+        stx <dos_scratch_vector
```

Comment:
There is a routine that setups the transfer area, and that also makes sure that it doesn't overlap hypervisor mapped memory, as otherwise feeding bad values into Y will result in being able to cause the process descriptor to be copied over aribtrary parts of the hypervisor.
I'll find the name of the function for you...

Reply:
```
jsr hypervisor_setup_copy_region
bcc fail
...
sta (<hypervisor_userspace_copy_vector),y
...
```